src/sdk/bucketspace_fs.js

```diff
@@ -247,6 +247,8 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             _.isUndefined(account.nsfs_account_config.gid)) return false;
         try {
             dbg.log0('_has_access_to_nsfs_dir: checking access:', ns.write_resource, account.nsfs_account_config.uid, account.nsfs_account_config.gid);
+            //TODO:  Operation not permitted error on change_user() with non root user, and 
```

Comment:
@naveenpaul1 can you explain what is the problem here? this should work with non root user